.github/workflows/jetbrains-updates.yml

```diff
@@ -10,20 +10,28 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v4
+            - name: Install dependencies
+              run: |
+                cd ./components/ide/jetbrains/image/gha-update-image
+                yarn
+                npm i -g bun
             - name: Check for updates
               run: |
-                  cd ./components/ide/jetbrains/image/gha-update-image
-                  yarn install
-                  node index.js
-                  git diff
+                cd ./components/ide/jetbrains/image/gha-update-image
+                node index.js
+                bun run update-versions.ts
```

Comment:
Sneaked bun in to the mix because of how useful it was with making this quick script (utilizing bun shell, io and typescript support). Hope we can use it for these smaller decoupled pieces more in the future.

Reply:
if it only for CI it is alright, but let's not spread it